docs/content/how-to/aws/create-infra-iam-separately.md

```diff
@@ -134,7 +135,118 @@ where
     to your workers. An ssh key will have been generated for you and stored as a secret in the 
     same namespace as the hosted cluster.
 
-Running this command should result in the following resources getting applied to your cluster:
+# Working with yaml
+## Creating the AWS infra & IAM
+
+Use the `hypershift create infra aws` command:
+
+    hypershift create infra aws --name CLUSTER_NAME \
+        --aws-creds AWS_CREDENTIALS_FILE \
+        --base-domain BASEDOMAIN \
+        --infra-id INFRA_ID \
+        --region REGION \
+        --render-yaml \  # Causes yaml to be rendered
+        --output-file OUTPUT_INFRA_FILE
+
+where
+
+* `CLUSTER_NAME` is the name of the hosted cluster you intend to create. This is used for creating
+    the Route53 private hosted zones that belong to the cluster.
+* `AWS_CREDENTIALS_FILE` points to an AWS credentials file that has permission to create
+    infrastructure resources for your cluster such as VPCs, subnets, NAT gateways, etc.
+    It should correspond to the AWS account for your guest cluster, where workers will live.
+* `BASEDOMAIN` is the base domain that will be used for your hosted cluster's ingress. It must
+    correspond to an existing Route53 public zone that you have access to create records in.
+* `INFRA_ID` is a unique name that will be used to identify your infrastructure via tags. It is used
+    by the cloud controller manager in Kubernetes and the CAPI manager to identify infrastructure
+    for your cluster. Typically this is the name of your cluster (CLUSTER_NAME) with a random
```

Comment:
What happens if this is set to be the same as CLUSTER_NAME or something totally different?

Reply:
Should make no difference, it can be the same or different, the only catch is that there isn't another HCI or run of the CLI with the same infra-id value.